activate.sh

```diff
@@ -5,8 +5,15 @@
 
 # Make sure that the controller node looks for the local pause image
 # TODO: remove this as soon as possible. As an idea, we could use a systemd drop-in unit.
+
+if hostnamectl | grep -q [Kk]ubic; then
```

Comment:
This seems brittle, what if I run CaaSP with a hostname of fookubicfoo - this will match. We'd be better off pulling dist info out of `lsb-release -a`.